library/alloc/src/vec/is_zero.rs

```diff
@@ -35,6 +37,47 @@ impl_is_zero!(char, |x| x == '\0');
 impl_is_zero!(f32, |x: f32| x.to_bits() == 0);
 impl_is_zero!(f64, |x: f64| x.to_bits() == 0);
 
+// Wrapping<T> and Saturating<T> are transparent over the T they contain, so if the T is zero
+// then the Wrapping or Saturating is also zero.
+// Manual u8 and i8 implementations are needed because of the specialization for u8 and i8
+// in spec_from_elem.
+
+unsafe impl IsZero for Wrapping<u8> {
+    fn is_zero(&self) -> bool {
```

Comment:
```suggestion
    #[inline]
    fn is_zero(&self) -> bool {
```

as all other impls in this file have it explicitly?

Reply:
Shouldn't matter since it's called from within std.

Reply:
That said I wouldn't be opposed to it.